pywinauto/__init__.py

```diff
@@ -89,7 +89,7 @@ def _get_com_threading_mode(module_sys):
     from . import findwindows
 
     WindowAmbiguousError = findwindows.WindowAmbiguousError
-    ElementNotFoundError = findwindows.ElementNotFoundError
+    WindowNotFoundError = findwindows.WindowNotFoundError
 
     if UIA_support:

```

Comment:
Honestly, not sure if  we still need this condition. Might worth removing.

Reply:
Hm, I guess such aliases are completely harmless, and removing them might break third-party packages which use find_window() and catch this exception... I'd highly suggest never breaking compatibility in minor versions, as per semantic versioning, since we have no idea of the impact on end users.

Reply:
I meant `if UIA_support` condition. We might just set ElementNotFoundError, ElementAmbiguousError uncoditionally.

Reply:
Whups, indeed now that you mention it, I don't find the relationship between UIA_support and these exceptions in nowadays's code... but the PR is already merged.